docs/install.md

```diff
@@ -1485,6 +1485,18 @@ deployment is in the `Failed` state, then one can look into the event(s) using
 > driver could run can be configured by using `nodeSelector` property of
 > [`DeploymentSpec`](#deployment-crd-api).
 
+#### Operator metrics data
+
+PMEM-CSI operator exposes below metrics data in addition to the
+[metrics data provided by the controller-runtime](https://book-v1.book.kubebuilder.io/beyond_basics/controller_metrics.html):
+
+Name | Type | Explanation
+-----|------|------------
+`pmem_csi_deployment_reconcile_started_at` | gauge | Timestamp at which a PmemCSIDeployment reconcile started by the operator. Labeled by the deployment name.
```

Comment:
Recording a timestamp seems to be driven by the intention to use this for E2E tests. But I cannot imaging how it could be useful for anything else, like a dashboard showing a timeline.

Wouldn't it be more useful to just count events, like `pmem_csi_deployment_reconcile_started`, `pmem_csi_deployment_sub_resource_created` and `pmem_csi_deployment_sub_resource_updated`?

Those then can be plotted over time. The E2E test then of course needs to be updated, but it can still determine when the operator did some work, so it should be doable.

Reply:
> Recording a timestamp seems to be driven by the intention to use this for E2E tests.

I was more referring to [operator-framework's CR metrics API](https://github.com/operator-framework/operator-lib/blob/main/handler/instrumented_enqueue_object.go) and tried to more align with that. The part of this API that was mostly driver by E2E test was `pmem_csi_deployment_sub_resource_updated_at` to address #943.

>

Reply:
Changed the PR such that now it records `pmem_csi_deployment_reconcile_{start,end}` counter vectors labeled with deployment name and `pmem_csi_deployment_sub_resource_{created,updated}_at`  gauge vectors with timestamps.